frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -230,58 +239,78 @@
           </div>
         </div>
         <div class="form-group">
-          <button class="btn btn-primary" type="submit" v-if="update">
-            {{ T.contestNewFormUpdateContest }}
-          </button>
-          <button class="btn btn-primary" type="submit" v-else="">
-            {{ T.contestNewFormScheduleContest }}
+          <button class="btn btn-primary" type="submit">
+            {{
+              update
+                ? T.contestNewFormUpdateContest
+                : T.contestNewFormScheduleContest
+            }}
           </button>
         </div>
       </form>
     </div>
   </div>
 </template>
 
+<style lang="scss">
+@import '../../../../../../node_modules/vue-multiselect/dist/vue-multiselect.min.css';
+
+.multiselect__tag {
+  background: #678dd7;
+}
+</style>
+
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import { omegaup } from '../../omegaup';
 import T from '../../lang';
 import DateTimePicker from '../DateTimePicker.vue';
+import Multiselect from 'vue-multiselect';
 
 @Component({
   components: {
     'omegaup-datetimepicker': DateTimePicker,
+    Multiselect,
   },
 })
 export default class NewForm extends Vue {
   @Prop() data!: omegaup.Contest;
   @Prop() update!: boolean;
+  @Prop() allLanguages!: string[];
+  @Prop() initialStartTime!: Date;
+  @Prop() initialFinishTime!: Date;
 
   T = T;
-  alias = this.data.alias;
-  availableLanguages = this.data.available_languages;
-  contest = this.data;
-  contestantMustRegister = this.data.contestant_must_register;
-  description = this.data.description;
-  feedback = this.data.feedback;
-  finishTime = this.data.finish_time;
-  scoreboard = this.data.scoreboard;
-  languages = this.data.languages;
-  needsBasicInformation = this.data.needs_basic_information;
-  penalty = this.data.penalty;
-  penaltyType = this.data.penalty_type;
-  penaltyCalcPolicy = this.data.penalty_calc_policy;
-  pointsDecayFactor = this.data.points_decay_factor;
-  requestsUserInformation = this.data.requests_user_information;
-  startTime = this.data.start_time;
-  showPenalty = this.data.show_penalty;
-  showScoreboardAfter = this.data.show_scoreboard_after;
-  submissionsGap = this.data.submissions_gap;
-  title = this.data.title;
+  alias = this.data?.alias ?? '';
```

Comment:
ahora que lo pienso, como `data` ni `alias` se están declarando como nulable, no podría ser esto

```suggestion
  alias = this.data.alias;
```

? igual con todas las demás cosas no-nulables.

pero después de una segunda revisión, resulta que `data` jamás se está estableciendo desde afuera! mejor hay que eliminar `data` para evitar confusión.

Reply:
Si se usa cuando se manda llamar desde el archivo `edit.js`, pero para que no haya confusión, mandaré los mismos datos que se mandan en el archivo `new.ts` y elimino `data`